browser/help/topics/v4-browser-hts.md

```diff
@@ -0,0 +1,245 @@
+  - `age_distribution`: Struct containing age distribution information for variant.
+    - `het`: Struct containing age distribution information for individuals heterozygous for this variant. Structured to allow easy histogram creation.
+      - `bin_edges`: Array containing the edges of each bin of the histogram.
+      - `bin_freq`: Array containing the frequency of individuals in this bin.
+      - `n_smaller`: Number of individuals with lower age than the lowest bin.
+      - `n_larger`: Number of individuals with a higher age than the highest bin.
+    - `hom`: Struct containing age distribution information for individuals homozygous for this variant. Structured to allow easy histogram creation. Contains same fields as `het` above.
+  - `filters`: Set containing variant QC filters. See `filters` description on the v4 Hail Tables [help page](v4-hts#filters).
+  - `quality_metrics`: Struct containing variant quality metric histograms information.
+    - `allele_balance`: Struct containing variant allele balance histograms information.
+      - `alt_adj`: Struct containing variant allele balance information calculated across high-quality genotypes. Contains same fields as other histogram structs. This data is displayed in the "Allele balance for heterozygotes" histogram in the browser's variant page.
+      - `alt_raw`: Struct containing variant allele balance information calculated across unadjusted genotypes. Contains same fields as other histogram structs.
+    - `genotype_depth`: Struct containing information used to display genotype depth (DP) histograms.
+      - `all_adj`: Struct containing DP information calculated using high-quality genotypes. Contains same fields as other histogram structs.
+      - `all_raw`: Struct containing DP information calculated across unadjusted genotypes. Contains same fields as other histogram structs.
+      - `alt_adj`: Struct containing DP information calculated using high-quality genotypes (variant carriers only). Contains same fields as other histogram structs.
+      - `alt_raw`: Struct containing DP information calculated across unadjusted genotypes (variant carriers only). Contains same fields as other histogram structs.
+    - `genotype_quality`: Struct containing information used to display genotype quality (GQ) histograms.
+      - `all_adj`: Struct containing GQ information calculated using high-quality genotypes. Contains same fields as other histogram structs.
+      - `all_raw`: Struct containing GQ information calculated across unadjusted genotypes. Contains same fields as other histogram structs.
+      - `alt_adj`: Struct containing GQ information calculated using high-quality genotypes (variant carriers only). Contains same fields as other histogram structs.
+      - `alt_raw`: Struct containing GQ information calculated across unadjusted genotypes (variant carriers only). Contains same fields as other histogram structs.
```

Comment:
on line 94 below, you have this description:
```
See `histograms` description on the v4 Hail Tables [help page](v4-hts#joint-histograms).
```
you could apply that here as well to save space